content/general/concepts/nft/nft.md

```diff
@@ -0,0 +1,29 @@
+---
+Title: 'NFT'
+Description: 'Non-fungible token which act like a digital representation to prove a ownership for a digital good backed by blockchain technology.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Development'
+Tags:
```

Comment:
Review [the "Tags" document](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) and only include tags from there. If you wish to add tags, please insert them in alphabetical order in the tags.md file and include the updated tags.md file in your PR.

If you're adding tags, "cryptocurrency" might be better than "NFT", as it's broader and might cover more entries.

Reply:
^^^^ 

@SSwiniarski I agree! We should definitely add "cryptocurrency" to `tags.md`. 👍

Reply:
Added Cryptocurrency, Blockchain, and Web3 tags to **tags.md**